crates/egui/src/widgets/text_edit/builder.rs

```diff
@@ -70,6 +70,7 @@ pub struct TextEdit<'t> {
     min_size: Vec2,
     align: Align2,
     clip_text: bool,
+    char_limit: Option<usize>,
```

Comment:
This could be `char_limit: usize` with a default of `usize::MAX` for simpler code

Reply:
Done